### PR TITLE
Small typos and redundancy

### PR DESCRIPTION
**Pull request by Riuqlav**
**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [x ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -6,22 +6,22 @@ issues.
 
 This policy is adapted from the policies of the following CNCF projects:
 
-* [Envoy](https://github.com/envoyproxy/envoy)
-* [CoreDNS](https://github.com/coredns/coredns)
+- [Envoy](https://github.com/envoyproxy/envoy)
```
Comment (travisn) [inline SECURITY.md:9]: Why change all the bullets from `*` to `-`? In the past we had some tooling that converted it to `*`, so i'm reluctant to see it changed again to `-`.

**Review by travisn — CHANGES_REQUESTED** [review]
See the DCO CI failure, please add a signature.

File: SECURITY.md (hunk 1 above)
Comment (Riuqlav) [reply SECURITY.md:9]: It's just Prettier editing it on save actually

Comment (Riuqlav) [reply SECURITY.md:9]: I am going into the file documentation now

**Review by subhamkrai — CHANGES_REQUESTED** [review]
please squash to a single commit. Thanks!

File: SECURITY.md (hunk 1 above)
Comment (travisn) [reply SECURITY.md:9]: @galexrt Just converted these a while back, so we need to make sure we're using consistent tooling so we don't keep changing them. If I change it to a `-` in vscode, I see the linter complain, so we wouldn't want these changes unless we change our tools:
```
MD004/ul-style: Unordered list style [Expected: asterisk; Actual: dash]markdownlint[MD004](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md004)
```

**Review by travisn — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (galexrt) [reply SECURITY.md:9]: @travisn I think there is an option in the `markdownlint` of VSCode to change the unordered list style, though I have to say we should commit to one as changing that option will cause these linter complaints in the unchanged documents.

I would prefer to stay with `*` and look to changing the `markdownlint` config for "everyone" working on Rook if possible.

**Pull request closed without merging**